back/config/cube/config/schema/Activities.js

```diff
@@ -0,0 +1,63 @@
+const tenantHost = COMPILE_CONTEXT.securityContext.tenant_host;
```

Comment:
This assumes that `tenant_host` has been added to the JWT token. We'll also have to find a way to do something similar for matomo where we'll need to pull out the `siteId` from somewhere.